srcpkgs/mailnag/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'mailnag'
+pkgname=mailnag
+version=2.2.0
+revision=1
+build_style=python3-module
+hostmakedepends="python3 gettext"
+# install gst-plugins-base1 if you want the Sound Notifications plugin
+# install libnotify and a libnotify client if you want the LibNotify Notifications plugin
+# a libsecret client (GNOME Keyring, Kwallet...) is needed
+# You should also have a working syslog (https://docs.voidlinux.org/config/services/logging.html) if you do not want error messages about syslog
```

Comment:
You can add `mailnag/files/README.voidlinux` file containing this info, and install it in `post_install` with `vdoc $FILESDIR/README.voidlinux`. Having it in the template isn't much help to users,

Reply:
I'm having problems with git, but I'll try to add those changes.

Reply:
Fixed.